CHANGELOG-5.2.md

```diff
@@ -9,6 +9,7 @@ To get the diff between two versions, go to https://github.com/symfony/symfony/c
 
 * 5.2.0-BETA2 (2020-10-14)
 
+ * feature #22592 [DoctrineBridge] Allow validating every class against unique entity constraint (wkania)
```

Comment:
This file is auto-generated from the titles of the merged PRs. No need to add an entry here.

Reply:
Fixed. Pull request template misguided me :)

Reply:
Ah, I see. There‘s a CHANGELOG.md file inside the Doctrine bridge directory. This is the one the PR template refers to in your case. 😃